lib/utils/dialogs/dialog.dart

```diff
@@ -0,0 +1,58 @@
+import 'package:flutter/material.dart';
+
+class DialogRequest extends StatefulWidget {
```

Comment:
Please move this to packages/nartus_ui_package/, create a new directory and name it 'dialog'

Reply:
done.